File: SalesDepot.Services/StatisticService/MainUserReportModel.cs
namespace SalesDepot.Services.StatisticService
{
	public partial class MainUserReportModel
	{
		public string FullName
		{
			get { return (firstName + " " + lastName).Trim(); }
		}

		public string GroupName
		{
			get { return !string.IsNullOrEmpty(group) ? string.Format("{0} ({1} {2})", group, groupUserCount, (groupUserCount > 1 ? "users" : "user")) : null; }
		}

		public double LoginsPercent
		{
			get { return groupLogins > 0 ? (double)userLogins / groupLogins : 0; }
		}

		public double DocsPercent
		{
			get { return groupDocs > 0 ? (double)userDocs / groupDocs : 0; }
		}

		public double VideosPercent
		{
			get { return groupVideos > 0 ? (double)userVideos / groupVideos : 0; }
		}

		public double TotalPercent
		{
			get { return groupTotal > 0 ? (double)userTotal / groupTotal : 0; }
		}
	}
}

File: WebSalesLibraries/protected/models/statistic/models/reports/MainUserReportModel.php
<?php

	/**
	 * Class MainUserReportModel
	 */
	class MainUserReportModel
	{
		/**
		 * @var string firstName
		 * @soap
		 */
		public $firstName;
		/**
		 * @var string lastName
		 * @soap
		 */
		public $lastName;
		/**
		 * @var string group
		 * @soap
		 */
		public $group;
		/**
		 * @var int groupUserCount
		 * @soap
		 */
		public $groupUserCount;
		/**
		 * @var int userTotal
		 * @soap
		 */
		public $userTotal;
		/**
		 * @var int groupTotal
		 * @soap
		 */
		public $groupTotal;
		/**
		 * @var int userLogins
		 * @soap
		 */
		public $userLogins;
		/**
		 * @var int groups
		 * @soap
		 */
		public $groupLogins;
		/**
		 * @var int groups
		 * @soap
		 */
		public $userDocs;
		/**
		 * @var int groups
		 * @soap
		 */
		public $groupDocs;
		/**
		 * @var int groups
		 * @soap
		 */
		public $userVideos;
		/**
		 * @var int groups
		 * @soap
		 */
		public $groupVideos;
	}